.github/workflows/TagBot.yml

```diff
@@ -1,11 +1,15 @@
 name: TagBot
 on:
-  schedule:
-    - cron: 0 * * * *
+  issue_comment:
+    types:
+      - created
+  workflow_dispatch:
 jobs:
   TagBot:
+    if: github.event_name == 'workflow_dispatch' || github.actor == 'JuliaTagBot'
     runs-on: ubuntu-latest
     steps:
       - uses: JuliaRegistries/TagBot@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          ssh: ${{ secrets.DOCUMENTER_KEY }}
```

Comment:
This is specifically needed when unit test CI is run on github action to work around the limitation that "one github action cannot trigger another one". MappedArrays is still using travis so we don't need this.

Most importantly, I believe for this repo, `DOCUMENTER_KEY` key isn't set.

Reply:
You're right, but do you plan to move away from travis at some point?

Reply:
I believe we should but given that this package isn't actively developed, this is of low priority. I don't have write access to this repo so we should leave @timholy to do so.